runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/Credentials/Credentials.state.ts

```diff
@@ -2,8 +2,17 @@ import { Credential } from 'runbotics-common';
 
 import { BasicCredentialDto } from '#src-app/views/credentials/Credential/Credential.types';
 
+interface AvailableCredentials {
+    id: string;
+    order: number;
+    credentialId: string;
+    processId: string;
+    credential: Credential;
+}
+
 export interface CredentialsState {
     all: BasicCredentialDto[];
     allByTemplateAndProcess: Credential[];
+    allAvailable: AvailableCredentials[];
```

Comment:
Sure, I've renamed it to AllProcessAssigned because these are all credentials assigned to a process in configure tab: 
![image](https://github.com/user-attachments/assets/d7f527e6-3a8e-4498-a567-f860ed5117af)

Reply:
basically what is in process_credential relation 
![image](https://github.com/user-attachments/assets/e0224ea0-6dfe-4c53-986d-9cb7ddcb41b5)

Reply:
Does the name make sense now?